app/controllers/api/v1/businesses_controller.rb

```diff
@@ -1,70 +1,110 @@
 module Api
-  module V1
-    class BusinessesController < ApplicationController
-      def index
-        @businesses = params[:user_id].present? ? Business.where(user_id: params[:user_id]).first : Business.all
-        render json: @businesses.with_attached_image
-      end
+	module V1
```

Comment:
Might want to configure your IDE to use 2 spaces for indents

Reply:
Gotcha Brad, I had to turn off prettier's `use tabs` option.